src/batches/setBatchSegmentationDetectLesion.m

```diff
@@ -0,0 +1,16 @@
+%-----------------------------------------------------------------------
```

Comment:
OK let's focus on this one for now.

I think we can then turn this into a function that returns the matlabbatch variable.

We actually have template to show you how it's supposed to look.

https://github.com/cpp-lln-lab/CPP_SPM/blob/master/src/templates/setBatchTemplate.m

Also to make our life easier: I would use our `getAnatFilename` to find the anatomical file we want to use for the 
`unified_segmentation.step1data` that is currently `UNDEFINED`.

https://github.com/cpp-lln-lab/CPP_SPM/blob/master/src/getAnatFilename.m